packages/cli-v3/src/build/entryPoints.ts

```
@@ -0,0 +1,128 @@
+    // Add required entry points
+    if (config.configFile) {
+      entryPoints.push(config.configFile);
+    }
+
+    if (target === "dev") {
+      entryPoints.push(...devEntryPoints);
+    } else {
+      entryPoints.push(...deployEntryPoints);
+    }
+
+    if (config.instrumentedPackageNames?.length ?? 0 > 0) {
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)